setup.py

```diff
@@ -0,0 +1,25 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='Housing Package',
+    version='0.1.0',
+    author='Saachi Mohanty',
+    author_email='saachi.mohanty@tigeranalytics.com',
+    description='A brief description of your package',
```

Comment:
Instead of `setup.py` can you switch to `pyproject.toml`..

Reply:
Created pyproject.toml file